clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -533,7 +533,11 @@ def err_stack_tagging_requires_hardware_feature : Error<
   "Armv9, try compiling with -march=armv8a+memtag or -march=armv9a+memtag">;
 
 def err_pauth_cpu_feature_missing : Error<
-  "A feature is requested that needs CPU with Pointer Authentication support">;
+  "neither FEAT_PAUTH nor -fptrauth-soft is enabled. Most of PAuth features are unavailable">;
```

Comment:
I already left a comment in clang/lib/Basic/Targets/AArch64.cpp and fixing that might make the current comment obsolete, but leaving it here anyway. It looks like that a **feature** is *enabled*, while an **option** is *specified* or *passed*. So, the following might be better: "neither FEAT_PAUTH is enabled nor -fptrauth-soft is passed..."

Reply:
Fixed